src/test/java/teammates/test/cases/browsertests/InstructorStudentRecordsPageUiTest.java

```diff
@@ -117,6 +118,38 @@ private void testLinks() {
         // TODO add link to a feedback session
     }
 
+    private void testFeedbackResponseCommentEditAndDeleteAction() {
+        InstructorAttributes instructor;
+        StudentAttributes student;
+
+        instructor = testData.instructors.get("teammates.test.CS2104");
+        student = testData.students.get("benny.c.tmms@ISR.CS2104");
+
+        instructorId = instructor.googleId;
+        courseId = instructor.courseId;
+        studentEmail = student.email;
+
+        viewPage = getStudentRecordsPage();
+
+        ______TS("Typical Case: Edit and add empty comment");
+
+        viewPage.editFeedbackResponseComment("-GRQ-1-1-1-1", "");
+        viewPage.verifyCommentFormErrorMessage(Const.StatusMessages.FEEDBACK_RESPONSE_COMMENT_EMPTY);
+        viewPage.closeEditFeedbackResponseCommentForm("-GRQ-1-1-1-1");
+
+        ______TS("Typical Case: Edit comment");
+
+        viewPage.editFeedbackResponseComment("-GRQ-1-1-1-1",
+                "Instructor first edited comment to Alice about feedback to Benny");
+        viewPage.verifyCommentRowContent("-GRQ-1-1-1-1", "Instructor first edited comment to Alice about feedback to Benny",
+                                            "teammates.test@gmail.tmt");
```

Comment:
Setting this to 16 spaces as setting it on 8  or 12 spaces gives checkstyle error.